packages/firestore/src/local/local_store.ts

```diff
@@ -1394,10 +1395,11 @@ export function hasNewerBundle(
 
 /**
  * Saves the given `BundleMetadata` to local persistence.
+ * @param bundleMetadata
```

Comment:
Missing description. You can also just remove this line.

Reply:
Done.